Firestore/Example/App/CMakeLists.txt

```diff
@@ -17,26 +17,26 @@ if(NOT APPLE)
 endif()
 
 add_executable(
-  firestore_example_app_macos MACOSX_BUNDLE
+  firebase_firestore_example_app MACOSX_BUNDLE
```

Comment:
Question: what is the rationale behind removing `macos` from the name?

Reply:
The idea is that our `objc_test` invocations will want to work on all supported platforms but they need to specify a host application in which to run. If I left the platform names in the app names, every `objc_test` would need a block like this above it:

```cmake
if(APPLE)
  if(IOS)
    set(host_app firestore_example_app_ios)
  else()
    set(host_app firestore_example_app_macos)
  endif()

  objc_test(
    firebase_firestore_objc_integration_test
    HOST ${host_app}
    ...
  )
endif()
```

Given that we can't actually build for macOS and iOS simultaneously changing this to a neutral name means this becomes:

```cmake
if(APPLE)
  objc_test(
    firebase_firestore_objc_integration_test
    HOST firebase_firestore_example_app
    ...
  )
endif()
```